build.js

```diff
@@ -48,4 +49,18 @@ html += '</body>\n</html>'
 
 var filename = path.resolve(__dirname, 'examples', 'index.html');
 fs.writeFileSync(filename, html);
-console.log(filename);
+
```

Comment:
Thanks. I definitively prefer this.

Can you group this together with creation of js/impress.js: Move this above line 37.